src/main/java/net/sf/jabref/importer/fileformat/MedlineImporter.java

```diff
@@ -84,50 +121,371 @@ public boolean isRecognizedFormat(BufferedReader reader) throws IOException {
     public ParserResult importDatabase(BufferedReader reader) throws IOException {
         Objects.requireNonNull(reader);
 
-        // Obtain a factory object for creating SAX parsers
-        SAXParserFactory parserFactory = SAXParserFactory.newInstance();
-
-        // Configure the factory object to specify attributes of the parsers it
-        // creates
-        parserFactory.setValidating(true);
-        parserFactory.setNamespaceAware(true);
-
-        // Now create a SAXParser object
         List<BibEntry> bibItems = new ArrayList<>();
         try {
-            SAXParser parser = parserFactory.newSAXParser(); // May throw
-            // exceptions
-            MedlineHandler handler = new MedlineHandler();
-            // Start the parser. It reads the file and calls methods of the
-            // handler.
-            parser.parse(new InputSource(reader), handler);
-
-            // Switch this to true if you want to make a local copy for testing.
-            if (false) {
-                reader.reset();
-                try (FileOutputStream out = new FileOutputStream(new File("/home/alver/ut.txt"))) {
-                    int c;
-                    while ((c = reader.read()) != -1) {
-                        out.write((char) c);
+            JAXBContext context = JAXBContext.newInstance("generated");
```

Comment:
Can you use a typed JAXBContext by passing the XXX.class, with XXX being the one with the `@XmlRootElement`?